### PR TITLE
SEO-176113-Xamarin-Page-With-Redirect4

### PR DESCRIPTION
**Pull request by berylouma**
Hello, 

I have worked on the links with redirects.

Regards,
Beryl.

### CODE REVIEW

File: Xamarin/Effects-View/Customization.md [hunk 1]
```diff
@@ -392,6 +392,6 @@ var effectsView = new SfEffectsView
 
 ## See also
 
-[How to get the selected item from ListView while using SfEffectsView within ItemTemplate](https://www.syncfusion.com/kb/11661/how-to-get-the-selected-item-from-listview-while-using-effectsview-within-itemtemplate)
+[How to get the selected item from ListView while using SfEffectsView within ItemTemplate](https://support.syncfusion.com/kb/article/10193/how-to-get-the-selected-item-from-listview-while-using-effectsview-within-itemtemplate?isInternalRefresh=False)
```
Comment (Yvone-Atieno) [inline Xamarin/Effects-View/Customization.md:395]: Check this link

File: Xamarin/Radio-Button/Event.md [hunk 2]
```diff
@@ -70,6 +70,6 @@ This demo can be downloaded from this [link](https://www.syncfusion.com/download
 
 ## See also
 
-[How to notify the selection changes in Xamarin.Forms radio button (SfRadioButton)](https://www.syncfusion.com/kb/11037/how-to-notify-the-selection-changes-in-xamarin-forms-radiobutton-sfradiobutton)
+[How to notify the selection changes in Xamarin.Forms radio button (SfRadioButton)](https://support.syncfusion.com/kb/article/9544/how-to-notify-the-selection-changes-in-xamarin-forms-radiobutton-sfradiobutton?isInternalRefresh=False)
```
Comment (Yvone-Atieno) [inline Xamarin/Radio-Button/Event.md:73]: Check this link

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: Xamarin/Radio-Button/Event.md (hunk 2 above)
Comment (berylouma) [reply Xamarin/Radio-Button/Event.md:73]: Done

File: Xamarin/Effects-View/Customization.md (hunk 1 above)
Comment (berylouma) [reply Xamarin/Effects-View/Customization.md:395]: Done

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Pull request merged**